articles/connections/database/custom-db/error-handling.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Learn how to handle errors when using your database as an identity provider.
+crews: crew-2
+topics:
+    - connections
+    - custom-database
+contentType: how-to
+useCase:
+    - customize-connections
+---
+# Custom Database Error Handling
+
+In this article, we will show you how you can handle errors resulting from your custom database connection.
```

Comment:
This article does not really say how to handle errors, all I can see is the different types of errors and how I can return one.